core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -4651,7 +4707,7 @@ private void checkLiteral2(String expression, String expected) {
     sql(query).ok(expected);
   }
 
-  @Test void testCubeInSpark() {
+  @Test public void testCubeWithGroupBy() {
```

Comment:
I remember that there was a PR that removed `public`. We'd better follow this style.

Reply:
> I remember that there was a PR that removed `public`. We'd better follow this style.

well, let me change it.